docs/apis-tools/zeebe-api/gateway-service.md

```diff
@@ -226,7 +226,7 @@ Only processes with none start events can be started through this command.
 :::
 
 :::note
-Start instructions have the same [limitations as the process instance modification](/components/concepts/process-instance-modification.md#limitations), e.g., it is not possible to start an element in a multi-instance subprocess.
+Start instructions have the same [limitations as process instance modification](/components/concepts/process-instance-modification.md#limitations), e.g., it is not possible to start an element at a sequence flow.
```

Comment:
>it is not possible to start an element at a sequence flow.

@christinaausley I think this is not correct. An element is not started at a sequence flow. Rather, a sequence flow is an element. 

IMO, `it is not possible to start at a sequence flow` is better. We actually mean to say `it is not possible to start a process instance at a sequence flow`